pkg/check/util.go

```diff
@@ -57,8 +60,12 @@ func toMySQLVersion(v string) (MySQLVersion, error) {
 	return version, nil
 }
 
-// IsAtLeast means v >= min
-func (v MySQLVersion) IsAtLeast(min MySQLVersion) bool {
+// Be means v >= min
+func (v MySQLVersion) Be(min MySQLVersion) bool {
```

Comment:
Is `Be` short for `bigger than or equal to`
it seems `ge` and `gt` are used more often

Reply:
I am definitely not good at writing shells.